.github/workflows/ai-revision.yaml

```diff
@@ -54,21 +61,50 @@ jobs:
           manubot_url=$(echo "$manubot_line" | awk -F"- " '{print $2}')
 
           pip install ${manubot_url}#egg=manubot[ai-rev]
+          
+          # install the multi-provider version of manubot-ai-editor over the current one, just for now
+          pip install git+https://github.com/manubot/manubot-ai-editor@langchain-anthropic-integration#egg=manubot-ai-editor
```

Comment:
Reminder to update this part.

Reply:
Removed as ccbc45f; it's now installed in the default way, via `manubot[ai-rev]`, which will pick up the latest `manubot-ai-editor` release from PyPI.